### PR TITLE
PAINTROID-444 Show size of shapes in different tools

### PR DESCRIPTION
**Pull request by JulianJautz**
Done for Clipboard-, Shape-, Import- and Text-Tool
Chip for current shape size is shown above other tool specific options
When shape is being resized, the current shape size is always shown now, 
independently of the toolOptionsLayout.

https://jira.catrob.at/browse/PAINTROID-444

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [x] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/DefaultTextToolOptionsView.kt [hunk 1]
```diff
@@ -74,6 +78,9 @@ class DefaultTextToolOptionsView(rootView: ViewGroup) : TextToolOptionsView {
         fontTypes = FontType.values().toList()
         initializeListeners()
         textEditText.requestFocus()
+        textToolOptionsViewShapeSizeChip = textToolView.findViewById<LinearLayout>(R.id.pocketpaint_layout_text_tool_options_view_shape_size).findViewById(R.id.pocketpaint_fill_shape_size_text)
```
Comment (Lenkomotive) [inline Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/DefaultTextToolOptionsView.kt:81]: Please try to fit those lines inside the "ruler" for readability.

File: Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/DefaultClipboardToolOptionsView.kt [hunk 2]
```diff
@@ -62,5 +91,11 @@ class DefaultClipboardToolOptionsView(rootView: ViewGroup) : ClipboardToolOption
         cutChip = stampToolOptionsView.findViewById(R.id.action_cut)
         enablePaste(false)
         initializeListeners()
+        stampToolOptionsView.run {
+            shapeSizeChip = findViewById<LinearLayout>(R.id.pocketpaint_layout_clipboard_tool_options_view_shape_size).findViewById(R.id.pocketpaint_fill_shape_size_text)
```
Comment (Lenkomotive) [inline Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/DefaultClipboardToolOptionsView.kt:95]: Please try to fit those lines inside the "ruler" for readability.

**Review by Lenkomotive — CHANGES_REQUESTED** [review]

**Comment by JulianJautz** [comment]
> I noticed a small bug for the tools Clipboard and Shape. When resizing slowly the bug doesnt occur but when I resize a little bit faster, it pushes the size information to the top of the settings as expected but leaves behind the size information where it was previously, [here is a video](https://github.com/Catrobat/Paintroid/assets/71790852/00ac606c-a5a1-41d5-9534-aea87bfd6d54)

This bug was due to the animation of the Chip. I removed the animation now, which fixes the issue (when the movement is too fast, the chip is set invisible before the animation is finished and it gets set visible at the end of the animation again). Drawback, the Chip is visible before the tool options vanish...

**Review by Lenkomotive — APPROVED** [review]
👍🏼

**Review by juliajulie95 — APPROVED** [review]

**Pull request merged**